data/json/recipes/recipe_food.json

```diff
@@ -9977,5 +9977,48 @@
       [ "water_clean", 2 ]
       ]
     ]
-  }
+  },
+  {
+  "type" : "recipe",
+  "result": "hflesh_pemmican",
+  "category": "CC_FOOD",
```

Comment:
Another good use for "copy-from" could start here (removing lines 9984-9988), but you'll need to add "autolearn: false" since it would otherwise inherit the autolearn: true from pemmican recipe which overrides book_learn

So something like this:
```json
    "result": "hflesh_pemmican",
    "copy-from": "pemmican",
    "autolearn": false,
    "book_learn": [[ "cookbook_human", 5 ]],

Reply:
Thank you for your help Draco